src/app/containers/create-movie/index.jsx

```diff
@@ -1,28 +1,22 @@
 'use client'
 
-import { useMutation } from "@tanstack/react-query"
-import axios from "axios"
-
-const createMovie = async () => {
-    const response = await axios.post('/api/movies', {id: 7, name: "Dune" })
-
-    return response.data
-}
+import CreateMovieForm from "./createMovieForm"
+import { useCreateMovie } from "../hooks/useCreateMovie";
+import DisplayError from "./displayError";
+import DisplaySuccess from "./displaySuccess";
 
 export const CreateMovie = () => {
-    const { mutate, data, isSuccess, isError, error } = useMutation({
-        mutationFn: createMovie, 
-        mutationKey: ["create-movie"]
-    })
-
-    return <>
-        {isError && <div className="text-red-600">{error.message}</div>}
-        <button onClick={() => mutate()}>Create movie</button>
-        {isSuccess && <div>
-            Newly created movie
-            <p>This is id: {data.id}</p>
-            <p>This is movie name: {data.name}</p>
-            </div>}
-    </>
+    const { mutate, data, isSuccess, isError, error } = useCreateMovie();
 
-}
\ No newline at end of file
+    const handleFormSubmit = (values) => {
```

Comment:
This is just a renaming for the mutate function. We could rename it where we destructure it from the useMutation

`const { mutate: handleFormSubmit, data, isSuccess, isError, error } = useCreateMovie()`

Reply:
Solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)